deployments/helm/hephaestus/templates/controller/cronjob.yaml

```diff
@@ -0,0 +1,78 @@
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: {{ include "hephaestus.manager.name" . }}-cronjob
+spec:
+  schedule: {{ .Values.controller.ibGarbageCollector.interval | quote }}
+  jobTemplate:
+    spec:
+      template:
+        metadata:
+          annotations:
+            checksum/config: {{ include (print $.Template.BasePath "/controller/secret.yaml") . | sha256sum }}
+            {{- if .Values.controller.metrics.enabled }}
+            prometheus.io/port: "{{ .Values.controller.manager.metricsPort }}"
+            prometheus.io/scrape: "true"
+            {{- end }}
+            {{- with .Values.controller.podAnnotations }}
+            {{- toYaml . | trimSuffix "\n" | nindent 8 }}
+            {{- end }}
+            {{- with .Values.podAnnotations }}
+            {{- toYaml . | nindent 8 }}
+            {{- end }}
+          labels:
+            {{- include "hephaestus.controller.labels.matchLabels" . | nindent 12 }}
+              {{- with .Values.controller.podLabels }}
```

Comment:
ultimately it doesn't matter to helm because you're using `{{- ... | nindent }}`, but I'd like the indent level to match between `matchLabels` and `podLabels`